docs/spark-profiling-tool.md

```diff
@@ -573,6 +645,55 @@ Usage: java -cp rapids-4-spark-tools_2.12-<version>.jar:$SPARK_HOME/jars/*
                         /path/to/eventlog2
 ```
 
+### Auto-Tuner support
+
+Starting with release "_22.10_", the Profiling tool a new "_Auto-Tuner_" that aims at optimizing
+Apache Spark applications by recommending a set of configurations to tune the performance of
+Rapids accelerator.  
```

Comment:
Add a sentence that gives details on how the auto tuner determines configs, e.g. that it takes as input both the cluster properties (if available) as well as the application event log to determine recommended configurations.